css/sign-In.css

```
@@ -0,0 +1,418 @@
+
+.Earnings__right-side {
+  border-radius: var(--eight-pixel);
+  text-align: center;
+  margin: 0px 25px;    
+}
+
+.Earnings__right-title{
+  font-size: var(--thirty-two);
+  font-weight: var(--font-weight-bold);
+  line-height: 90px;
+  letter-spacing: var(--letter-spacing-negative);
+  color: var(--color-black-small);
+  margin: 0px;
+}
+
+.Earnings__right-text {
+  font-size: var(--eighteen-pixel);
+  font-weight: var(--font-weight);
+  line-height: var(--thirty-two);
+  color: var(--color-striking-gray);
+  margin-left: 0px;
+  margin-bottom: 34px;
+  margin-top: var(--zero);    
+}
+
+.Earnings__right-button {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 28px;
+  gap: var(--font-size-xlarge);
+  flex-direction: column;
+  margin-left: 25px;
+  margin-right: 30px;
+}
+
+.Earnings__right-img{
+  color: var(--color-black-medium);
+  font-size: var(--sixteen-pixeles);
+  font-weight: var(--font-weight-small);
+  line-height: var(--line-height-medium);
+  margin-right: 8px;
+}
+
+.Earnings__right-apple,
+.Earnings__right-google {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  color: var(--color-black-medium);
+  background-color: var(--color-white);
+  box-shadow: var(--box-shadowx);
+  margin: var(--zero);
+  font-size: var(--sixteen-pixeles);
+  font-weight: var(--font-weight-small);
+  line-height: var(--line-height-medium);
+  border: 1px solid var(--color-light-gray);
+  padding: 8px 26px;
+  border-radius: var(--eight-pixel);
+}
+
+.Earnings__right-google:hover {
+  background-color: var(--color-solid);
+}
+
+.Earnings__right-apple:hover {
+  background-color: var(--color-solid);
+}
```

Si es lo mismo, juntalos
```suggestion
.Earnings__right-google:hover,
.Earnings__right-apple:hover {
  background-color: var(--color-solid);
}
```